src/app/(landing)/@about/about.module.scss

```diff
@@ -0,0 +1,27 @@
+@use '../../../styles/colors';
+@use '../../../styles/mixins';
+
+.frame {
+    background-color: colors.getColor(secondary, 50);
+  }
+
+  .container {
+    margin-right: auto;
+    position: relative;
+    padding: 12rem 0;
+    z-index: 1;
+    justify-content: center;
+    align-items: center;
+  }
+  
+  .yellow {
+    color: colors.getColor(warning, 400);
+  }
+
+  .textBox{
```

Comment:
Use the Flex component to position instead - The text component just needs an align text

Reply:
done